modules/ob-about-buildrun.adoc

```diff
@@ -0,0 +1,17 @@
+// Ths module is included in the following assembly:
+//
+// * builds/overview-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="about-buildrun_{context}"]
+= About build run
```

Comment:
Because the content is so specific to the resource itself, suggest modifying the title to the following [object reference form](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) (but since it is a heading, no backticks):

```suggestion
= About the BuildRun resource
```